src/app/pages/join-session/join-session.component.html

```diff
@@ -10,7 +10,14 @@
         </mat-form-field>
         <mat-form-field appearance="outline" class="form-field">
           <mat-label>Name</mat-label>
-          <input matInput class="name"  formControlName="nameControl">
+          <input matInput class="name" formControlName="nameControl">
+        </mat-form-field>
+        <mat-form-field appearance="outline" class="form-field">
+          <mat-label>Nest Barcode</mat-label>
+          <input matInput inputmode="numeric" pattern="[0-9]*" class="nest" formControlName="nestControl">
+          <mat-error *ngIf="sessionFormGroup.controls['nestControl'].invalid">
+            This number is from 20 to 100 and can be found on the printed nest marker in the upper left corner
```

Comment:
I don't think we should include the range here, that is subject to change and doesn't actually matter to the students. Maybe something like "Invalid nest ID: please enter the ID number on your nest marker".

Reply:
Also it should be noted that currently the range of possible barcodes is actually 0-63 but I think we will be switching to a different type of barcode. See #124.